profiles/common/modules/custom/tmgmt_poetry/inc/tmgmt_poetry.webservice.inc

```diff
@@ -186,158 +212,225 @@ function FPFISPoetryIntegrationRequest($user, $password, $msg) {
             $message = t('No message.');
           }
 
-          $message_job->addMessage("DGT update received. Request status: @status. Message: @message", array(
-            '@status' => $status_message,
-            '@message' => $message,
-          ));
+          $message_job->addMessage(
+            "DGT update received. Request status: @status. Message: @message",
+            [
+              '@status' => $status_message,
+              '@message' => $message,
+            ]
+          );
+
           if ($cancelled) {
             $reference = '%' . $reference;
-            $ids = _tmgmt_poetry_obtain_related_translation_jobs(array(), $reference)->fetchAll();
+
+            $ids = _tmgmt_poetry_obtain_related_translation_jobs([], $reference)
+              ->fetchAll();
             foreach ($ids as $id) {
               $job = tmgmt_job_load($id->tjid);
-              $job->aborted("Request aborted by DGT.", array());
+              $job->aborted('Request aborted by DGT.', []);
```

Comment:
The function "aborted" is in the file "tmgmt.entity.job.inc" (http://api.worldempire.ch/api/tmgmt/entity!tmgmt.entity.job.inc/7-1).
In the platform, the patch is "/profiles/multisite_drupal_standard/modules/contrib/tmgmt/entity/tmgmt.entity.job.inc".

This function does the classic setState of the Job and the setState of associated job Item.